libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount.m

```diff
@@ -29,17 +29,18 @@
 #import "SFCommunityData.h"
 #import "SFIdentityData.h"
 
-static NSString * const kUser_ACCESS_SCOPES     = @"accessScopes";
-static NSString * const kUser_CREDENTIALS       = @"credentials";
-static NSString * const kUser_EMAIL             = @"email";
-static NSString * const kUser_FULL_NAME         = @"fullName";
-static NSString * const kUser_ORGANIZATION_NAME = @"organizationName";
-static NSString * const kUser_USER_NAME         = @"userName";
-static NSString * const kUser_COMMUNITY_ID      = @"communityId";
-static NSString * const kUser_COMMUNITIES       = @"communities";
-static NSString * const kUser_ID_DATA           = @"idData";
-static NSString * const kUser_CUSTOM_DATA       = @"customData";
-static NSString * const kUser_IS_GUEST_USER     = @"guestUser";
+static NSString * const kUser_ACCESS_SCOPES       = @"accessScopes";
+static NSString * const kUser_CREDENTIALS         = @"credentials";
+static NSString * const kUser_EMAIL               = @"email";
+static NSString * const kUser_FULL_NAME           = @"fullName";
+static NSString * const kUser_ORGANIZATION_NAME   = @"organizationName";
+static NSString * const kUser_USER_NAME           = @"userName";
+static NSString * const kUser_COMMUNITY_ID        = @"communityId";
+static NSString * const kUser_COMMUNITIES         = @"communities";
+static NSString * const kUser_ID_DATA             = @"idData";
+static NSString * const kUser_CUSTOM_DATA         = @"customData";
+static NSString * const kUser_IS_GUEST_USER       = @"guestUser";
+static NSString * const kUser_ACCESS_RESTRICTIONS = @"guestUser";
```

Comment:
It's one of the properties on the SFUserAccount.h header.  An `SFUserAccountAccessRestriction` enum